zcash_primitives/Cargo.toml

```diff
@@ -8,10 +8,12 @@ authors = [
 [dependencies]
 byteorder = "1"
 ff = { path = "../ff" }
+hex = "0.3"
```

Comment:
Sigh! [Rust should have added a hex literal.](https://github.com/rust-lang/rfcs/pull/2244)